### ISSUE TITLE
Version numbers mess

### ISSUE THREAD

**Issue by ashep** [cosmtrek/air#155]
Hello!

If I look at GitHub, I see that latest release is 1.15.1. If I look at version number at `air` start I see 1.12.1. Finally, if i look at `go list -m` after `go get -u github.com/cosmtrek/air`, I am getting 1.21.2. How to determine version number I've installed?

Thank you!

**Comment by visoft** [comment]
v1.15.1 is the latest version, I installed with `go get -u github.com/cosmtrek/air@v1.15.1` Without the version number, you get 1.21.2 which is a typo, I believe. Hopefully it gets fixed and a new version is released soon.

**Comment by cosmtrek** [comment]
Fixed with new release version v1.25

**Comment by ashep** [comment]
Thank you for the update. Nevertheless I am unable to install a new version.

```
go get -u github.com/cosmtrek/air
~/go/bin/air -v

  __    _   ___
 / /\  | | | |_)
/_/--\ |_| |_| \_ v1.12.1 // live reload for Go apps, with Go1.14.0
```

```
go get -u github.com/cosmtrek/air@v1.25
go get github.com/cosmtrek/air@v1.25: no matching versions for query "v1.25"
```

**Comment by xiantang** [comment]
> Thank you for the update. Nevertheless I am unable to install a new version.
> 
> ```
> go get -u github.com/cosmtrek/air
> ~/go/bin/air -v
> 
>   __    _   ___
>  / /\  | | | |_)
> /_/--\ |_| |_| \_ v1.12.1 // live reload for Go apps, with Go1.14.0
> ```
> 
> ```
> go get -u github.com/cosmtrek/air@v1.25
> go get github.com/cosmtrek/air@v1.25: no matching versions for query "v1.25"
> ```

i released v1.26.0.
plz use  go get  -u github.com/cosmtrek/air to update

**Comment by ashep** [comment]
@xiantang thank you!
